### ISSUE TITLE
Documentation has broken link

### ISSUE THREAD

**Issue by reagle** [getpelican/pelican#2282]
I'm trying to figure out how to configure smart and note that in the [docs](http://docs.getpelican.com/en/stable/settings.html) the link in the text is broken: "Refer to the Python Markdown documentation’s Options section for a complete list of supported options." 

<http://pythonhosted.org/Markdown/reference.html#markdown> is 404.

**Comment by Wangjunyu** [comment]
On Page [Writing content](https://github.com/getpelican/pelican/blob/ce38e318bd4905f75c988ee930620ebcd65588c0/docs/content.rst) the following sentense also have an broken link :
`Pelican also supports Markdown Extensions, which might have to be installed separately if they are not included in the default Markdown package and can be configured and loaded via the MARKDOWN setting.`

**Comment by reagle** [comment]
I ran a link checker quickly on the page I report and found:

```
Line: 364 http://pythonhosted.org/Markdown/reference.html

Line: 1136 http://feeds.example.com/
Status: 500 Can't connect to feeds.example.com:80
This is a server side problem. Check the URI.

error Line: 1463 http:///
Status: 500 No Host option provided
This is a server side problem. Check the URI.

error Line: 995 http://msdn.microsoft.com/en-us/library/cdax410z(VS.71).aspx redirected to https://www.microsoft.com/en-us/download/details.aspx?id=55979
```

**Comment by p1v0t** [comment]
Another broken link on [writing content](http://docs.getpelican.com/en/3.7.1/content.html) page:
For Markdown, which utilizes the __CodeHilite extension__ to provide syntax highlighting ...

**Comment by reagle** [comment]
A link checker should be run on all the content...

**Comment by ingwinlu** [comment]
I only see the problem with python-markdown's documentation being moved without redirect. should be a simple pr to fix and point those links towards https://python-markdown.github.io/.

> Line: 1136 http://feeds.example.com/
:shipit:

**Comment by justinmayer** [comment]
Anyone in this thread want to submit a PR that updates the relevant links?

**Comment by jorgesumle** [comment]
This has already been solved with #2302

**Comment by justinmayer** [comment]
@jorgesumle / @jonathan-s: Thanks for the pointer!
